services/core-api/app/api/mines/permits/nod/resources/nod.py

```diff
@@ -0,0 +1,24 @@
+from flask_restplus import Resource, marshal_with, inputs
+from werkzeug.exceptions import BadRequest, NotFound
+from flask_restplus import reqparse
+from app.extensions import api
+from app.api.utils.resources_mixins import UserMixin
+from app.api.utils.access_decorators import (requires_any_of, VIEW_ALL, MINESPACE_PROPONENT)
+from app.api.mines.permits.nod.models.nod import Nod
+from app.api.mines.permits.permit.models.permit import Permit
+from app.api.mines.mine.models.mine import Mine
+from app.api.mines.response_models import PERMIT_NOD_MODEL
+
+
+class NodResource(Resource, UserMixin):
```

Comment:
if these are being split out into 3 seperate resource files I am just wondering should the Notice of departure by NoD guid be under permits or at a global level like mines or should this just be a part of the permit one?

Reply:
I think putting it under permits made sense to me as it relates to a departure from what is stipulated within a permit. It's always tricky to know the right place to have it. but I feel notice_of_departure is its own module that is directly linked to permits, which is why I put it within the permits folder.

Reply:
Having said that, as the feature grows, we can always move stuff around in the future